README.md

```diff
@@ -1,96 +1,112 @@
-PLoud Validator version 1.1
-==================
+# PLoud Validator
 
 PLoudValidator is a software to validate audio files with correct loudness.
 It can validate various specifications:
 
 * EBU R 128
-
 * ATSC A85
-
 * CST-Ficam RT-017 specification for France.
 
 It can also correct file to EBU R 128.
 Comming soon for others specifications.
 
-License
--------
+
+## License
 
 See [**COPYING.md**](COPYING.md)
 
-Compilation
------------
 
-* Getting the source
+## Compilation
+
+#### Getting the source
 
-> git clone git://github.com/mikrosimage/loudness_validator.git  
+```
+git clone git://github.com/mikrosimage/loudness_validator.git  
 cd loudness_validator
+```
+
+#### Compilation
+
+###### Build all tools in release mode.
+```
+scons
+```
+
+###### Build in debug mode.
+```
+scons mode=debug
+```
+
+###### Build with a specific path to libraries.
+```
+scons BOOST_ROOT=/path/to/boost SNDFILE_ROOT=/path/to/sndfile
+```
 
-* Linux compilation
 
-> export BOOST_ROOT=path/to/boost  
-export QTDIR=/usr/lib64/qt4  
-./build_linux.sh  
-./build_linux.sh /library /tools /PloudValidator  
+## Run applications
 
-see ./build_linux.sh --help for more informations
+#### Analyser
+```
+build/release/loudness-analyser
+```
 
-* Windows compilation
+Analyse the loudness of the given file.
 
-> set BOOST_ROOT=path/to/boost  
-set QTDIR="C:\QtSDK\mingw" # with default install  
-set QTDIR="C:\QtSDK\Desktop\Qt\4.7.4\mingw"  
-build_windows.bat  
-build_windows.bat  /library /tools /PloudValidator  
+#### Corrector
+```
+build/release/loudness-corrector
+```
 
-see ./build_windows.bat --help for more informations
+Correct the loudness of the given file.
 
-Run applications
-----------------
+#### validator
+```
+build/release/loudness-validator
+```
 
-The distribution is done into dist/release/ directory.  
-You can run GUI with:  
->./dist/release/bin/PLoudValidator  
+A GUI which enables to analyse, visualize and correct the loudness.
 
-Or using command line tools, to display help :  
->./dist/release/bin/loudness-analyser  
-./dist/release/bin/loudness-corrector  
 
-Tested compilers
-----------------
+## Tested compilers
 
 * GCC 4.4.1 on OpenSUSE 11.2
+* G++ 4.6.2 on OpenSUSE 11.2
```

Comment:
I did compile with g++...
But we can call it “GCC” as a common shorthand term for the GNU Compiler Collection.
See this link:
https://gcc.gnu.org/onlinedocs/gcc-3.3.5/gcc/G_002b_002b-and-GCC.html